app/editor/src/components/modal/styled/Modal.tsx

```diff
@@ -38,7 +38,7 @@ export const Modal = styled(Col)`
     margin: 10% auto;
     border-radius: 3px;
     min-width: fit-content;
-    max-width: 750px;
+    max-width: 85%;
```

Comment:
@jdtoombs Is this going to cause issues?  I personally don't like percentages as they cause all kinds of resizing challenges, but sometimes they're the right way to go.